lib/sequelize.js

```diff
@@ -22,6 +22,7 @@ const Association = require('./associations/index');
 const Validator = require('./utils/validator-extras').validator;
 const Op = require('./operators');
 const deprecations = require('./utils/deprecations');
+const { default: PQueue } = require('p-queue');
```

Comment:
Minor refactor (code style preference)

```suggestion
const PQueue = require('p-queue').default;
```